release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -570,6 +570,13 @@ See this for example (1.20 was the "future release"):
 * 1.18 https://github.com/kubernetes/website/pull/25390
 * 1.19 https://github.com/kubernetes/website/pull/25467
 
+Changes required:
+- Set `latest` to the [future release] version
+- Update `fullversion` and `githubbranch` to the latest patched version of the corresponding release
+- Update the list of versions to include the [future release] and to remove the oldest release
+- Set `deprecated` to `true`
+- Update the patch version for all past release to the latest patch version available
```

Comment:
nit:  Update the patch version for all past releases to the latest patch version available
Is there an example of this? Where do you get the patch information (check the GitHub tags?)?
Before:
```
[[params.versions]]
fullversion = "v1.19.0"
version = "v1.19"
githubbranch = "v1.19.0"
. . .

```

After:
```
[[params.versions]]
fullversion = "v1.19.4"
version = "v1.19"
githubbranch = "v1.19.4"
. . . 
```